File: SantanderTeste/app/src/main/java/com/example/alessandrofsouza/santanderapp/service/ApiService.java
package com.example.alessandrofsouza.santanderapp.service;

import com.example.alessandrofsouza.santanderapp.model.ContactModel;
import com.example.alessandrofsouza.santanderapp.model.InvestmentModel;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {

    public static final String BASE_URL = "https://floating-mountain-50292.herokuapp.com";

    @GET("/cells.json")
    Call<ContactModel> listCells();

    @GET("/fund.json")
    Call<InvestmentModel> listInvestment();

}

File: app/src/main/java/com/anabhomasi/androidapp/data/services/APIService.kt
package com.anabhomasi.androidapp.data.services

import com.anabhomasi.androidapp.data.models.Form
import com.anabhomasi.androidapp.data.models.Fund
import retrofit2.Call
import retrofit2.http.GET

interface APIService {

    @GET("cells.json")
    fun getCells(): Call<Form.Response>

    @GET("fund.json")
    fun getFunds(): Call<Fund.Response>
}